socialhome/streams/app/stores/streamStore.state.js

```diff
@@ -2,21 +2,37 @@ import _get from "lodash/get"
 
 
 export default function () {
+    const content = _get(window, ["context", "content"], undefined)
     const contentIds = []
     const unfetchedContentIds = []
     const contents = {}
+    if (content) {
+        content.replyIds = []
+        content.shareIds = []
+        contentIds.push(content.id)
+        contents[content.id] = content
+    }
```

Comment:
Why not use `contents` here and just define some `singleContentId` variable? This `singleContentId` can be replaced by a router later.

Reply:
Yeah, good point, fixed.